language/e2e_tests/src/tests/event_key.rs

```diff
@@ -0,0 +1,21 @@
+use crate::{account::Account, executor::test_all_genesis};
+use types::account_config::AccountEvent;
+
+#[test]
+fn stable_event_key() {
```

Comment:
This is not a good test for event key stability because the genesis transaction is special/unique in that it:
- has txn.sender == association_address
- calls create_account(association_address)

I think what you want is an ordinary transaction sent by `B` that calls `create_account(A)` where `A != B` and checks that all the handles correspond to `A` instead of `B`.